lib/components/user/monitored-trip/saved-trip-list.js

```diff
@@ -86,7 +86,13 @@ class TripListItem extends Component {
           <Panel.Title>
             <TripHeader>{trip.tripName}</TripHeader>
             <small>
-              From <b>{from.name}</b> to <b>{to.name}</b>
+              <FormattedMessage
+                id="components.SavedTripList.fromTo"
+                values={{
+                  from: <strong>{from.name}</strong>,
```

Comment:
Is this allowed? Didn't we have an intl-specific strong tag?

Reply:
100% allowed, and it simplifies the string resource.